exaudfclient/base/build.sh

```diff
@@ -1 +1,3 @@
-bazel build $*
\ No newline at end of file
+#!/bin/bash
+
+bazel build "$@"
```

Comment:
Reason:
"                    ^-- SC2048: Use "$@" (with quotes) to prevent whitespace problems.
                    ^-- SC2086: Double quote to prevent globbing and word splitting."

Reply:
Should be fine